lib/phoenix_live_view.ex

```diff
@@ -399,6 +399,17 @@ defmodule Phoenix.LiveView do
         {:noreply, assign(socket, :temperature, new_temp)}
       end
 
+  | Binding                | Attributes |
+  |------------------------|------------|
+  | [Params](#module-click-events) | `phx-value-*` |
+  | [Click Events](#module-click-events) | `phx-click` |
+  | [Focus/Blur Events](#module-focus-and-blur-events) | `phx-blur`, `phx-focus`, `phx-target` |
+  | [Form Events](#module-form-events) | `phx-change`, `phx-submit`, `data-phx-error-for`, `phx-disable-with` |
+  | [Key Events](#module-key-events) | `phx-keydown`, `phx-keyup`, `phx-target` |
+  | [Rate Limiting](#module-rate-limiting-events-with-debounce-and-throttle) | `phx-debounce`, `phx-throttle` |
+  | [Custom DOM Patching](#module-custom-dom-patching) | `phx-update` |
+  | [JS Interop](#module-js-interop-and-client-controlled-dom) | `phx-hook` |
+
```

Comment:
what a great idea! 💯 Do you have a screenshot of how this looks after the docs are built?

Reply:
![image](https://user-images.githubusercontent.com/30214/65349871-72e85080-dbb2-11e9-95ca-84ba6e3b54ce.png)